### ISSUE TITLE
Scan tool trunkates large numbers

### ISSUE THREAD

**Issue by Jinotad** [ldtteam/Structurize#664]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Are you using the latest Structurize Version?

- [X] I am running the latest **alpha** version of Structurize for my Minecraft version.
I am also running the latest versions of other mods that are part of my problem.

### Did you check on the Wiki? or ask on Discord?

- [X] I checked the MineColonies/Structurize [Wiki](https://wiki.minecolonies.ldtteam.com/) or I asked on [discord](https://discord.minecolonies.com/).

### Minecraft Version

1.20

### Structurize Version

structurize-1.20.1-1.0.725-BETA.jar

### Current Behavior

The scan tool does not properly display large numbers of a single item.  If it tries to display 100,000 it would display 10,000 instead.
![2024-04-19_07 15 27](https://github.com/ldtteam/Structurize/assets/85356635/4a1e6d47-9909-4886-ae23-230c35b4e9ba)

![2024-04-19_07 16 00](https://github.com/ldtteam/Structurize/assets/85356635/9b8349fe-19ae-483f-a777-5bbc21f91ce7)

### Expected Behavior

Properly display values, or add abbreviations to indicate it's too large to display exactly.

### Reproduction Steps

Scan an 85 x 32 x 87 block area and check the display.

### Logs

None

### Footer

---
## Viewers
* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Nightenom** [comment]
Yeah, I'm aware of this, the problem is that block names are fighting the counts, either we need to rearrange the line in list or add rolling text parameter to blockui
